paper-radio-button.html

```diff
@@ -209,10 +209,8 @@
         }
       },
 
-      // create the element ripple inside the `radioContainer`
-      _createRipple: function() {
+      ready: function() {
```

Comment:
Yeah, but then it runs on every attachment. The cases where create is run without attached immediately after (it's not hard-coded into a template or something) seem like they'd be more likely to attach / detach multiple times.